x-pack/legacy/plugins/ml/public/components/chart_tooltip/chart_tooltip_service.d.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { BehaviorSubject } from 'rxjs';
+
+import { TooltipValue, TooltipValueFormatter } from '@elastic/charts';
+
+interface TooltipData {
+  name: string;
+  value: any;
```

Comment:
`string | ReactElement`?

Reply:
Fixed to reuse the types provided by Elastic Charts in a270966 (However, Elastic Charts treat this as `any` internally).